### PR TITLE
Clean up syntax

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: packages/wallet-sdk/src/connection/WalletLinkConnectionCipher.ts [hunk 1]
```diff
@@ -7,91 +7,83 @@ export class WalletLinkConnectionCipher {
   // @param secret hex representation of 32-byte secret
   constructor(private readonly secret: string) {}
```
Comment (bangtoven) [inline packages/wallet-sdk/src/connection/WalletLinkConnectionCipher.ts:9]: turn on `hide whitespace`
https://github.com/coinbase/coinbase-wallet-sdk/pull/1060/files?diff=split&w=1

image

File: packages/wallet-sdk/src/CoinbaseWalletSDK.ts [hunk 2]
```diff
@@ -3,7 +3,6 @@
 
 import { LogoType, walletLogo } from './assets/wallet-logo';
 import { DiagnosticLogger } from './connection/DiagnosticLogger';
-import { EventListener } from './connection/EventListener';
```
Comment (bangtoven) [inline packages/wallet-sdk/src/CoinbaseWalletSDK.ts:6]: turn on hide whitespace
https://github.com/coinbase/coinbase-wallet-sdk/pull/1060/files?diff=split&w=1

File: packages/wallet-sdk/src/CoinbaseWalletSDK.ts [hunk 3]
```diff
@@ -3,7 +3,6 @@
```
Comment (bangtoven) [inline packages/wallet-sdk/src/CoinbaseWalletSDK.ts:3]: turn on hide whitespace
https://github.com/coinbase/coinbase-wallet-sdk/pull/1060/files?diff=split&w=1

File: packages/wallet-sdk/src/relay/WalletLinkRelay.ts [hunk 4]
```diff
@@ -133,17 +121,19 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
...
+    const connection = new WalletLinkConnection({
       session,
-      this.linkAPIUrl,
-      this.listener,
-      this.diagnostic
-    );
+      linkAPIUrl,
+      diagnostic,
+      listener: this,
+    });
```
Comment (willn-cb) [inline packages/wallet-sdk/src/relay/WalletLinkRelay.ts:130]: 👍

File: packages/wallet-sdk/src/relay/WalletLinkRelay.ts [hunk 5]
```diff
@@ -133,17 +121,19 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
...
 
+    const { version, darkMode } = this.options;
     const ui = this.options.uiConstructor({
-      linkAPIUrl: this.options.linkAPIUrl,
-      version: this.options.version,
-      darkMode: this.options.darkMode,
+      linkAPIUrl,
+      version,
+      darkMode,
```
Comment (willn-cb) [inline packages/wallet-sdk/src/relay/WalletLinkRelay.ts:136]: @bangtoven if we're not using `version` and `darkMode` anywhere else why don't we just:
```
version: this.options.version
```
?

Comment (bangtoven) [reply packages/wallet-sdk/src/relay/WalletLinkRelay.ts:136]: @willn-cb i think i did that for the consistency and code readability. open to suggestions.

Comment (bangtoven) [reply packages/wallet-sdk/src/relay/WalletLinkRelay.ts:136]: image

Comment (bangtoven) [reply packages/wallet-sdk/src/relay/WalletLinkRelay.ts:136]: without git diffing, it actually looks like this ^ @willn-cb

Comment (willn-cb) [reply packages/wallet-sdk/src/relay/WalletLinkRelay.ts:136]: Yeah fair that's probably more readable

**Review by willn-cb — APPROVED** [review]
Looks good

**Pull request merged**